src/tests/evm_test.ts

```diff
@@ -84,6 +101,19 @@ test('Contract Calls', async t => {
       }
     }
 
+    if (rtv) {
+      let rProto = await client.txReceiptAsync(rtv)
+      if (rProto) {
+        let receipt = EvmTxReceipt.deserializeBinary(bufferToProtobufBytes(rProto))
+        t.deepEqual(
+          Array.prototype.slice.call(receipt.getContractaddress_asU8(), 0),
```

Comment:
A `Uint8Array` has a `slice()` method why are you calling `Array.prototype.slice` here?